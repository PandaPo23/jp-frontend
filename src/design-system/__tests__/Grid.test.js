import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Grid from '../Grid';

describe('Grid component', () => {
  it('should render as Grid', () => {
    const wrapper = mount(<Grid />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
