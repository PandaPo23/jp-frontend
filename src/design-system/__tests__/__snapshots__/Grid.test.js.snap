// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid component should render as Grid 1`] = `
.emotion-0 {
  display: grid;
}

<Grid
  display="grid"
>
  <div
    className="emotion-0"
  />
</Grid>
`;
